.travis.yml

```diff
@@ -0,0 +1,34 @@
+language: php
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
+matrix:
+  fast_finish: true
+  include:
+    - php: 5.3
+      dist: precise
+    - php: 5.4
+    - php: 5.5
+    - php: 5.6
+    - php: 7.0
+    - php: 7.1
+
+before_install:
+  - phpenv config-rm xdebug.ini
+  - composer self-update
+
+install:
+  - composer install
+
+script:
+  # this run phpunit/tests
+  - ./vendor/bin/phpunit
+  # this checks that the source code follows Code Syntax rules
+  # @todo update for Symfony v4
+  #- ./vendor/bin/phpcs --bootstrap=.php_cs
+
+  # this checks that the composer.json and composer.lock files are valid
+  - composer validate --strict
```

Comment:
Please add a newline at the end of configuration and PHP files to make viewing/editing files in the CLI easier.